### DISCUSSION TITLE
Where should diagnostics be placed for NorESM2.5 development simulations?

### DISCUSSION THREAD

**Discussion by mvertens** [NorESMhub/noresm3_dev_simulations#12] [Q&A] [answered]
Should there be a standard diagnostic place for diagnostic plots? If so - where?

**Comment by adagj** [comment]
Didn't we already make a common directory? If not I suggest to use NS9560K datalake. @TomasTorsvik @yanchun (I'm not sure why I can tag Yanchun...) what do you think?

**Reply by gold2718** [reply]
> @yanchun (I'm not sure why I can tag Yanchun...)

I think you mean GitHub ID YanchunHe (I think yanchun is some other user) and I think he does not auto-complete because he is not listed in the noresm_core group which is the group that has access to this repo.

**Comment by TomasTorsvik** [comment]
@adagj , @mvertens  - We have 4.2 TB storage left on NS9560K datalake, and are over the limit on the NS9560K datapeak partition. There is now a `/nird/datalake/NS9560K/www` directory, and I just created a `diagnostics` subdirectory under this, with group permission set to inherit the `ipcc` group (same as for `/nird/datapeak/NS2345K/www/diagnostics`). Output should be available from web pages under
`https://ns9560k.web.sigma2.no/datalake/diagnostics/`

I have set the group permission to `ipcc` since this was used before. Alternatively we could use `ns9560k` or `noresm`. I don't know who is associated with the different groups on Nird.

**Reply by adagj** [reply]
Thanks!

**Comment by oyvindseland** [comment]
It looks to me that it is around 100 Tb free on datapeak?

Project      Resource  Type    Usage       Soft Limit   Hard Limit               
NS9560K      datapeak  Disk    780.890 TiB  909 TiB      909 TiB    
NS9560K      datapeak  Files   11605469     16809984     19529728

**Reply by TomasTorsvik** [reply]
Yes, you are right. I looked at the wrong dusage log.

**Comment by TomasTorsvik** [comment]
@adagj , @mvertens , @oyvindseland - Since we already have some diagnostics output on `/nird/datapeak/NS9560K/www/diagnostics`, maybe it makes sense to continue using this?

**Reply by adagj** [reply]
Yes, I agree. Let's use the already existing one. Thanks for checking!

**Comment by adagj** [comment, marked as answer]
Answer to this discussion: please use the folder `/nird/datapeak/NS9560K/www/diagnostics` for diagnostics.

Output can be view from [https://ns9560k.web.sigma2.no/datapeak/diagnostics/](https://ns9560k.web.sigma2.no/datapeak/diagnostics/).
